.registry/resources/minimalaws.ui-schema.yaml

```diff
@@ -1,4 +1,4 @@
-version: 0.3
+version: v0.2.0
```

Comment:
I don't think this is the way that this ui-schema version is supposed to be used. I think it's intent is to be the version of the schema this file is using, not the version of the stack itself.

Reply:
it looks like we have a couple discrepancies with the ui schema files in our template stacks, they aren't up to date with the latest schema/design.  I've opened https://github.com/crossplane/crossplane/issues/1297 to track this.  currently assigned to @muvaf but @suskin could take this also, depending on who is least loaded :)

Reply:
> I think it's intent is to be the version of the schema this file is using, not the version of the stack itself.

@jbw976 Do you mean it is similar to `apiVersion` in the sense that the version given here indicates the version of the structure of this file that it wants to be parsed or just author's own versioning?